project_creator.bat

```diff
@@ -0,0 +1,10 @@
+@echo off

```

Comment:
Later this could be moved this from a separate file to the spl.bat as a new command (as in git).

spl.bat shall be a common entry point. Such as `git` the common entry point for all git commands is.

The command shall be a verb, not a noun.

```
$ spl create-project ...
```

So you could create a new project from within an existing project. But this is a new aspect to the first one.